src/targets/maven.ts

```diff
@@ -102,48 +93,115 @@ export class MavenTarget extends BaseTarget {
     );
   }
 
-  private getEnvVarOrDefault(envVar: string): string {
+  private getEnvVarOrDefault(envVar: OptionalConfig): string {
     if (process.env[envVar]) {
       return process.env[envVar] as string; // `as string` to make TS happy
     }
     return DEFAULT_ENV_VARIABLES[envVar];
   }
 
-  public async publish(version: string, _revison: string): Promise<void> {
-    await withTempDir(
-      async dir => {
-        const git = simpleGit(dir);
-        const username = await getAuthUsername(this.github);
-        this.githubRemote.setAuth(username, getGithubApiToken());
-        await git.clone(this.githubRemote.getRemoteStringWithAuth(), dir);
-        await git.checkout(`release/${version}`); // TODO: release name should be customized
-
-        await this.createUserGradlePropsFile();
-        const mavenReleaser = new MavenReleaser(
-          this.mavenConfig.distributionsPath,
-          this.mavenConfig.settingsPath,
-          this.mavenConfig.mavenRepoUrl,
-          this.mavenConfig.mavenRepoId,
-          this.mavenConfig.mavenCliPath,
-          this.mavenConfig.gradleCliPath
-        );
-        mavenReleaser.release();
-
-        await git.add(FILES_TO_COMMIT);
-        await git.commit(`craft(maven): Deployed ${version} to Maven Central.`);
-        if (this.shouldPush()) {
-          await git.push();
+  public async publish(_version: string, _revison: string): Promise<void> {
```

Comment:
If you are not using `_version` and `_revision`, something feels very off to me>

Reply:
This comment still applies.